Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -835,6 +841,59 @@ extension MapboxMap {
                                  featureId: featureId,
                                  stateKey: stateKey)
     }
+
+}
+
+// MARK: - View Annotations
+
+extension MapboxMap {
+
+    // TODO: Add documentation
+    public func setViewAnnotationPositionsUpdateListenerFor(listener: ViewAnnotationPositionsListener) {
+        __map.setViewAnnotationPositionsUpdateListenerFor(listener)
```

Comment:
What's the resulting ownership graph look like? Does this create a cycle if `listener` owns a strong reference to `MapboxMap`? If so, how do we break that cycle in `ViewAnnotationManagerImpl`?

Reply:
Good question, I think this could have been a retain cycle, but after the review change we only pass `DelegatingViewAnnotationPositionsListener` to it which only has a weak reference to the annotation manager. I tested it and it seems it's not leaking.
Also I've added an extra call to the destructor of the manager the clear the listener after we don't need it anymore.